src/box/lua/config/instance_config.lua

```diff
@@ -1694,7 +1694,98 @@ return schema.new('instance_config', schema.record({
                 end
             end,
         }),
-    })
+    }),
+    audit_log = enterprise_edition(schema.record({
+        to = enterprise_edition(schema.enum({
```

Comment:
Nit: I would comment `audit_log.to` regarding lack of `box_cfg` and `box_cfg_nondefault` annotations like it is done for `log.to`.